src/digitalIdentity-frontend/src/app/app-routing.module.ts

```diff
@@ -11,14 +10,18 @@ import { DIOverviewComponent } from './pages/DI/DIOverview-page/DIOverview-page.
 import { SchemaOverviewComponent } from './pages/schema/schema-overview-page/schema-overview-page.component';
 import { ErrorPageComponent } from './pages/error-page/error-page.component';
 import { ChangePasswordComponent } from './pages/change-password-page/change-password-page.component';
+
 import { CredDefOverviewPageComponent } from './pages/credential/credential-overview-page/credDef-overview-page.component';
+
 import { ProofTemplateOverviewPageComponent } from './pages/proofTemplate/proofTemplate-overview-page/proofTemplate-overview-page.component';
+import {CreateCreDefComponent} from "./pages/credential/create-creDef/create-creDef.component";
 
 const appRoutes: Routes = [
   { path: 'login', component: LoginPageComponent },
   { path: 'create-new-DI', component: CreateDIPageComponent },
   { path: 'DI-Overview', component: DIOverviewComponent },
   { path: 'create-schema', component: CreateSchemaPageComponent },
+  { path: 'create-credential', component: CreateCreDefComponent},
```

Comment:
'create-credential-definition'

Reply:
Please name the component "CreateCredDefComponent" everywhere (-> rename symbol), because the other components about credential definitions also use the abbreviation "CredDef"